EKF/control.cpp

```diff
@@ -84,6 +84,10 @@ void Ekf::controlFusionModes()
 	_mag_data_ready = _mag_buffer.pop_first_older_than(_imu_sample_delayed.time_us, &_mag_sample_delayed);
 	_baro_data_ready = _baro_buffer.pop_first_older_than(_imu_sample_delayed.time_us, &_baro_sample_delayed);
 
+	// update range sensor angle parameters in case they have changed
```

Comment:
I would move them into the setRangeData function inside the 

if (time_usec - _time_last_range > _min_obs_interval_us)

conditional so they are not unnecessarily calculated every EKF update.